src/wp-includes/media.php

```diff
@@ -5564,3 +5577,61 @@ function wp_increase_content_media_count( $amount = 1 ) {
 
 	return $content_media_count;
 }
+
+/**
+ * Retrieves an attachments preview image or icon source.
+ *
+ * @since n.e.x.t
```

Comment:
`n.e.x.t` does not work for WP core. Please use the actual targeted version (`6.1.0`). Also in other instances in this PR.